main.cpp

```diff
@@ -1,26 +1,59 @@
 #include "server/Server.hpp"
 #include <vector>
 #include "parsers/ConfigParse.hpp"
+#include <signal.h>
+
+bool endserver = false;
+
+void WebKill(int signal) {
+	if (signal == 2) {
```

Comment:
Using `SIGINT` instead of 2 would be more readable

Reply:
yes, done